### PR TITLE
worked in temp fixes from agent20

### PR DESCRIPTION
**Pull request by raajb**
@bryanp @rjclardy   

https://github.com/metabahn/agent20/commit/b329fb3d684a04d9b811974bc5e9c5b58cf2c0b1

### CODE REVIEW

**Comment by rjclardy** [comment]
👍  looks good to me

File: src/core/node.js [hunk 1]
```diff
@@ -482,7 +482,11 @@ pw.node = {
   },
 
   toA: function (nodeSet) {
-    return Array.prototype.slice.call(nodeSet);
+    if (nodeSet) {
+      return Array.prototype.slice.call(nodeSet);
+    } else {
+     return [];
+     }
```
Comment (bryanp) [inline src/core/node.js]: I think the indentation is wrong here.

Comment (raajb) [inline src/core/node.js]: okay fixed it

**Comment by raajb** [comment]
@bryanp is this good to go?

**Comment by bryanp** [comment]
Code looks solid. Please add a description of the fix to `toA` to `CHANGELOG.md`.

**Comment by rjclardy** [comment]
@raajb Here's the description you requested, feel free to modify:

  Fixes `toA` so that it returns an empty array instead of erroring out when `nodeSet` is missing

**Comment by raajb** [comment]
Alright @bryanp added a description

**Comment by bryanp** [comment]
Thanks @raajb :)

**Pull request merged**